packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -228,6 +229,13 @@ export const run = (includeInternalScripts = false) => {
       });
     });
 
+  program
+    .command('plugin:update')
+    .description('Update plugin')
```

Comment:
update seems too general.... maybe `update-circleci`?

Reply:
What about `npx grafana-toolkit plugin:update --circleci` ? I was trying to think about it quite generally.

Reply:
Like if you wanted to 'update' a plugin, one of the things might be circleci. But you might have a filter `--filter=circleci`